falcon/request.py

```diff
@@ -1865,11 +1866,13 @@ class RequestOptions:
             certain cases, such as when working with authentication
             schemes that employ URL-based signatures.
 
-        default_media_type (str): The default media-type to use when
-            deserializing a response. This value is normally set to the media
-            type provided when a :class:`falcon.App` is initialized; however,
-            if created independently, this will default to the
-            ``DEFAULT_MEDIA_TYPE`` specified by Falcon.
+        default_media_type (str): The default media-type used to
```

Comment:
While updating the references to include ASGI, I took the liberty of adding additional clarity to these descriptions.